android/src/main/kotlin/com/sap/gigya_flutter_plugin/GigyaSDKWrapper.kt

```diff
@@ -1063,6 +1082,141 @@ class GigyaSDKWrapper<T : GigyaAccount>(application: Application, accountObj: Cl
 
     //endregion
 
+    //region BIOMETRIC
+
+    fun biometricIsAvailable(channelResult: MethodChannel.Result) {
+        channelResult.success(sdkBiometric.isAvailable)
+    }
+
+    fun biometricIsLocked(channelResult: MethodChannel.Result) {
+        channelResult.success(sdkBiometric.isLocked)
+    }
+    
+    fun biometricIsOptIn(channelResult: MethodChannel.Result) {
+        channelResult.success(sdkBiometric.isOptIn)
+    }
+
+    fun biometricOptIn(arguments: Any, channelResult: MethodChannel.Result) {
+        val argumentMap = arguments as Map<*, *>;
+        sdkBiometric.optIn(activity, GigyaPromptInfo(argumentMap["title"] as String?,
+            argumentMap["subtitle"] as String?,
+            argumentMap["description"] as String?),
+        object : IGigyaBiometricCallback {
+            override fun onBiometricOperationSuccess(action: GigyaBiometric.Action) {
+                channelResult.success(null)
+            }
+
+            override fun onBiometricOperationFailed(reason: String?) {
+                // TODO: Update to use error code once updated on Gigya Android SDK
+                if (reason != "Fingerprint recognition failed") {
+                    channelResult.error(
+                        GENERAL_ERROR,
+                        "Operation Failed: $reason",
```

Comment:
Could you just use the reason here? Then it is in line with the other uses you have.

```suggestion
                        reason,
```